experiment_prototype/experiment_prototype.py

```diff
@@ -149,63 +119,121 @@
-    in kHz.
+    in kHz. **Not currently supported.**
 
-txfreq
+txfreq *required or clrfrqrange or rxfreq required*
     transmit frequency, in kHz. Note if you specify clrfrqrange it won't be used.
 
-rxfreq
+rxfreq *required or clrfrqrange or txfreq required*
     receive frequency, in kHz. Note if you specify clrfrqrange or txfreq it won't be used. Only
     necessary to specify if you want a receive-only slice.
 
-wavetype
-    string for wavetype. The default is SINE. Any other wavetypes not currently supported but
-    possible to add in at later date.
 
-iwavetable
-    a list of numeric values to sample from. The default is None. Not currently supported
-    but could be set up (with caution) for non-SINE.
+**Defaultable Slice Keys**
 
-qwavetable
-    a list of numeric values to sample from. The default is None. Not currently supported
-    but could be set up (with caution) for non-SINE.
+acf *defaults*
+    flag for rawacf and generation. The default is False. If True, the following fields are
+    also used:
+    - averaging_method (default 'mean')
+    - xcf (default True if acf is True)
+    - acfint (default True if acf is True)
+    - lagtable (default built based on all possible pulse combos)
+    - range_sep (will be built by pulse_len to verify any provided value)
 
-seqoffset
-    offset in us that this slice's sequence will begin at, after the start of the sequence.
-    This is intended for PULSE interfacing, when you want multiple slice's pulses in one sequence
-    you can offset one slice's sequence from the other by a certain time value so as to not run both
-    frequencies in the same pulse, etc.
-
-comment
-    a comment string that will be placed in the borealis files describing the slice.
+acfint *defaults*
+    flag for interferometer autocorrelation data. The default is True if acf is True, otherwise
+    False.
 
-averaging_method
-    a string defining the type of averaging to be done. Current methods are 'mean' or 'median.'
+averaging_method *defaults*
+    a string defining the type of averaging to be done. Current methods are 'mean' or 'median'.
     The default is 'mean'.
 
-acf
-    flag for rawacf and generation. The default is False.
+comment *defaults*
+    a comment string that will be placed in the borealis files describing the slice. Defaults
+    to empty string.
 
-xcf
-    flag for cross-correlation data. The default is True if acf is True, otherwise False.
+lag_table *defaults*
+    used in acf calculations. It is a list of lags. Example of a lag: [24, 27] from
+    8-pulse normalscan. This defaults to a lagtable built by the pulse sequence
+    provided. All combinations of pulses will be calculated, with both the first pulses
+    and last pulses used for lag-0.
 
-acfint
-    flag for interferometer autocorrelation data. The default is True if acf is True, otherwise
-    False.
+pulse_phase_offset *defaults*
+    Allows phase shifting between pulses, enabling encoding of pulses. Default all
```

Comment:
This may be misleading, since it's possible to have many different phase changes within each pulse
```suggestion
    Allows phase encoding within pulses. Default all
```

Reply:
I think my suggestion can be improved upon as well... maybe you have some better ideas